packages/lab/src/accordion/Accordion.css

```diff
@@ -81,8 +81,8 @@
   color: var(--accordion-summary-text-color);
   font-size: var(--uitkAccordion-summary-fontSize, var(--uitk-text-fontSize));
   font-family: var(--uitkAccordion-summary-fontFamily, var(--uitk-text-fontFamily));
-  padding-left: var(--uitkAccordion-summary-paddingLeft, var(--accordion-summary-paddingLeft));
```

Comment:
@james-nash I took a look at this and it actually affects the width of the icon so may be best to leave it for now